app/models/post.rb

```diff
@@ -1,5 +1,9 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: 'author_id', counter_cache: :posts_counter
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  belongs_to :author, class_name: 'User', counter_cache: :posts_counter
```

Comment:
- [ ] You did a great job working on the suggested changes! 💯 However, I'm having the same problem with the counters as the previous reviewer. The `posts_counter` is getting incremented twice on post creation. This is because you have a `counter_cache` and also an `after_save` hook. When a `Post` is created, these two functions increment the counter, ending up with double the value expected. Please consider removing this counter cache! 🤓 🔧 

```suggestion
  belongs_to :author, class_name: 'User'
```